index.js

```diff
@@ -0,0 +1,73 @@
+import { timeDate, date } from './modules/date.js';  // eslint-disable-line
+import * as nav from './modules/nav.js';  // eslint-disable-line
+
+const bookListEl = document.querySelector('#book-list');
+const addBtn = document.querySelector('.btn');
+const titleEl = document.querySelector('#title');
+const authorEl = document.querySelector('#author');
+
+const bookList = JSON.parse(localStorage.getItem('storedBooks')) || [];
+
+class Library {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+
+  addBook() {
+    addBtn.addEventListener('click', () => {
+      const title = titleEl.value;
+      const author = authorEl.value;
+      if (title && author) {
+        const addNewBook = {
+          title,
+          author,
+        };
+        bookList.push(addNewBook);
+        localStorage.setItem('storedBooks', JSON.stringify(bookList));
+        this.renderBooks();
+        // form.reset();
+        titleEl.value = '';
+        authorEl.value = '';
+      }
+    });
+  }
```

Comment:
- [ ] For the sake of best practice, kindly create a module for the Library class and then import its methods here.